contrib/log/slog/slog.go

```diff
@@ -28,6 +31,16 @@ type group struct {
 	attrs []slog.Attr
 }
 
+type config struct {
+	log128bits bool
+}
+
+func newConfig() *config {
+	return &config{
+		log128bits: os.Getenv("DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED") != "false",
```

Comment:
So it seems that `DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED` is `false` by default, but is there any case where it could be mistakenly set to something else? We have a function for getting boolean envvars (`instrumentation/options/GetBoolEnv()`) that lets you pass in a default value (for if the data is missing or if the data isn't a boolean as expected).